grudge/reductions.py

```diff
@@ -197,8 +196,13 @@ def nodal_min_loc(dcoll: DiscretizationCollection, dd, vec) -> float:
                    for idx in np.ndindex(vec.shape))
 
     actx = vec.array_context
-    return reduce(lambda acc, grp_ary: actx.np.minimum(acc, actx.np.min(grp_ary)),
-                  vec, np.inf)
+
+    # FIXME: do not force result onto host
+    return reduce(
+            lambda acc, grp_ary: np.minimum(
```

Comment:
```suggestion
            lambda acc, grp_ary: actx.np.minimum(
```
?

Reply:
Done in this force push: https://github.com/inducer/grudge/compare/335d27cfc84f222795693d8d6d90bfab21be0247..8131936e38f02b590261198529c8d167362b9d39